internal/bodyprocessors/raw.go

```diff
@@ -0,0 +1,40 @@
+package bodyprocessors
+
+import (
+	"io"
+	"strconv"
+	"strings"
+
+	"github.com/corazawaf/coraza/v3/experimental/plugins/plugintypes"
+	"github.com/corazawaf/coraza/v3/internal/collections"
+)
+
+type rawBodyProcessor struct {
+}
+
+func (*rawBodyProcessor) ProcessRequest(reader io.Reader, v plugintypes.TransactionVariables, _ plugintypes.BodyProcessorOptions) error {
+	buf := new(strings.Builder)
+	if _, err := io.Copy(buf, reader); err != nil {
```

Comment:
```suggestion
	var buf strings.Builder
	if _, err := io.Copy(&buf, reader); err != nil {
```

Generally can't be sure of eliding heap allocations but worth trying